src/main/java/practice/CandidateValidator.java

```diff
@@ -1,5 +1,27 @@
 package practice;
 
-public class CandidateValidator {
+import java.util.function.Predicate;
+import model.Candidate;
+
+public class CandidateValidator implements Predicate<Candidate> {
     //write your code here
+    private static final int PERIOD = 10;
+    private static final int MIN_AGE = 35;
+    private static final String SYMBOL = "-";
+    private static final int FROM = 0;
+    private static final int TO = 1;
+    private static final String NATION = "Ukrainian";
+
+    @Override
+    public boolean test(Candidate candidate) {
+        String[] split1 = candidate.getPeriodsInUkr().split(SYMBOL);
+        int period = Math.abs(Integer.parseInt(split1[FROM]) - Integer.parseInt(split1[TO]));
+        if (candidate.isAllowedToVote() && candidate.getAge() >= MIN_AGE
+                && candidate.getNationality().equals(NATION)
+                && period > PERIOD) {
+            return true;
+        }
+        return false;
```

Comment:
```suggestion
        return candidate.isAllowedToVote() 
                && candidate.getAge() >= MIN_AGE
                && candidate.getNationality().equals(NATION)
                && period > PERIOD;
```
you can simplify  it